### PR TITLE
Update docs for Restate 1.1.0

### PR DESCRIPTION
**Pull request by pcholakov**
Bumped Restate and Go SDK versions in config, and re-ran `generate.sh`.

### CODE REVIEW

**Comment by pcholakov** [comment]
@slinkydeveloper I'll hold off merging this until the latest SDK releases are done.

**Comment by slinkydeveloper** [comment]
You can merge on main i think

**Review by gvdongen — APPROVED** [review]

**Comment by slinkydeveloper** [comment]
Can you run the pre-release script again for java sdk 1.1? It should update the javadocs.

File: restate.config.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "RESTATE_VERSION": "1.0",
+  "RESTATE_VERSION": "1.1",
   "TYPESCRIPT_SDK_VERSION": "1.3.0",
   "JAVA_SDK_VERSION": "1.0.1",
```
Comment (slinkydeveloper) [inline restate.config.json:4]: this is now 1.1

**Comment by pcholakov** [comment]
Updated for Restate 1.1, Java SDK 1.1, and Python SDK 0.3.

File: docs/develop/rust/overview.mdx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+---
+
+import Admonition from '@theme/Admonition';
+
+# Overview
+
+The Restate Rust SDK is open source and can be found on GitHub: ([sdk-python repo](https://github.com/restatedev/sdk-rust)).
+
+<Admonition type="info" icon="🚧" title="Work in progress">
+The Rust SDK documentation is still under construction. Please refer to the [restate-sdk crate](https://docs.rs/restate-sdk/latest/restate_sdk/) to get started.
```
Comment (slinkydeveloper) [inline docs/develop/rust/overview.mdx:13]: Add a link to the hello world example too: https://github.com/restatedev/examples/tree/main/templates/rust

Comment (pcholakov) [reply docs/develop/rust/overview.mdx]: Done!

**Pull request merged**